frontend/web/src/app/components/stl-viewer/stl-viewer.component.ts

```diff
@@ -24,18 +30,22 @@ export class StlViewerComponent implements OnInit {
     stlLoader.setDecimalPrecision(3);
 
     // =========================================================================
-    this.viewer = new Viewer(canvas, parameters);
+    this.viewer = new Viewer(this.canvas, parameters);
     this.viewer.setLoader(stlLoader);
     this.viewer.setParameter('Renderer', 'webgl');
     this.viewer.init();
   }
 
-  resetView(){
+  resetView() {
     this.viewer.resetScene();
     this.viewer.update();
   }
 
-  toogleRotation(){
+  enlargeViewer() {
+    this.router.navigate([this.router.url + '/'+this.digitalPartID+'/stl']);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Use a template literal instead of concatenating with a string literal. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=typescript%3AS3512)